src/DeviceInput/deviceInputSystem.ts

```diff
@@ -355,17 +355,29 @@ export class DeviceInputSystem implements IDisposable {
                 pointer[PointerInput.DeltaVertical] = evt.movementY;
 
                 if (this.onInputChanged) {
+                    // The browser might use a move event in case
+                    // of simultaneous mouse buttons click for instance. So
+                    // in this case we stil need to propagate it.
+                    let fireFakeMove = true;

```

Comment:
does it work if you click AND move simulteanously? Don't we swallow one in this case?

Reply:
not that I have seen so far let me double check

Reply:
nope all good